proto/agent/v1/connection.proto

```diff
@@ -52,8 +54,20 @@ enum ConnectionUpdateSource {
     CONNECTION_UPDATE_SOURCE_HEURISTIC = 3;
 }
 
+message AgentConfig {
+    string dashboards = 1;
+    string alert_notifications = 2;
+}
+
+message CloudConfig {
+    string dashboards = 1;
+    string alert_notifications = 2;
+}
+
 message Config {
-    bool bearer_protection = 1;
-    bool cloud_only_notifications = 2;
-    bool universal_dashboards = 3;
+    bool bearer_protection = 1 [deprecated=true];
```

Comment:
[minor] I think deprecating fields like [this](https://github.com/netdata/cloud-schemas/blob/ff9a8dd82f9e46b90f1804aec0be4ad40c025d64/proto/alarm/v1/dispatcher.proto#L126) is better as it's completely removed from the Go files transformed from protos.

Reply:
makes total sense, fixed